relay-kafka/src/config.rs

```diff
@@ -156,8 +167,7 @@ impl Default for TopicAssignments {
             outcomes: "outcomes".to_owned().into(),
             outcomes_billing: None,
             sessions: "ingest-sessions".to_owned().into(),
-            metrics: "ingest-metrics".to_owned().into(),
-            metrics_sessions: None,
+            metrics_sessions: "ingest-metrics".to_owned().into(),
```

Comment:
Now there is duplication here between in default and the "alias" values, which I would like to avoid. Not sure the best way to tackle this.

Reply:
I think that's fine.